staging/src/k8s.io/kubectl/pkg/cmd/describe/describe.go

```diff
@@ -116,20 +116,20 @@ func (flags *DescribeFlags) ToOptions(parent string, args []string) (*DescribeOp
 	}
 
 	if len(args) == 0 && cmdutil.IsFilenameSliceEmpty(flags.FilenameOptions.Filenames, flags.FilenameOptions.Kustomize) {
-		return nil, fmt.Errorf("You must specify the type of resource to describe. %s\n", cmdutil.SuggestAPIResources(parent))
+		return nil, fmt.Errorf("you must specify the type of resource to describe. %s\n", cmdutil.SuggestAPIResources(parent))
 	}
 
 	builderArgs := args
 
 	describer := func(mapping *meta.RESTMapping) (describe.ResourceDescriber, error) {
-		return describe.DescriberFn(flags.Factory, mapping)
+		return describe.DescriberFn(flags.RESTClientGetter, mapping)
 	}
 
 	o := &DescribeOptions{
 		Selector:          flags.Selector,
 		Namespace:         namespace,
 		Describer:         describer,
-		NewBuilder:        flags.Factory.NewBuilder,
+		NewBuilder:        f.NewBuilder,
```

Comment:
The reason I used`factory.NewBuilder()` because it is an interface whereas `resource.NewBuilder` is a concrete function, which mean if you initialize the factory as a TestFactory instead of a normal Factory, then f.Builder() can be implemented and behave differently.

That's why I thougt `factory.NewBuidler()` is better

Also this will change the complete Test suite as well right if I'm not wrong ?